### PR TITLE
feat: move-to available on class context

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
Purpose of this PR is to allow move to action on test-takers classes.

We will separate move to action from single resource operation.

https://oat-sa.atlassian.net/browse/AUT-1008

### CODE REVIEW

**Review by pnal — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: actions/structures.xml [hunk 1]
```diff
@@ -39,7 +39,10 @@
                     <action id="testtaker-move" name="Move" url="/taoTestTaker/TestTaker/moveInstance" context="instance" group="none" binding="moveNode">
                         <icon id="icon-move-item"/>
                     </action>
-                    <action id="testtaker-move-to" name="Move To" url="/taoTestTaker/TestTaker/moveResource" context="resource" group="tree" binding="moveTo">
+                    <action id="testtaker-move-to" name="Move To" url="/taoTestTaker/TestTaker/moveResource" context="instance" group="tree" binding="moveTo">
+                        <icon id="icon-move-item"/>
+                    </action>
+                    <action id="class-move-to" name="Move To" url="/taoTestTaker/TestTaker/moveClass" context="class" group="tree" binding="moveTo">
```
Comment (mike-ionut-mihai-sandu-tao) [inline actions/structures.xml:45]: by binding the 2 actions to the same JS action are we creating a possible bug ?

Comment (bartlomiejmarszal) [reply actions/structures.xml:45]: We did, this is why in structures I changed context to separate those. This way we avoid this issue. Yet we want to use this same js binding to preserve similar ux feeling.

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Comment by AntonNeviarouski** [comment]
QA verified

**Pull request merged**